README.md

```diff
@@ -3,6 +3,7 @@
 Propel2 is an open-source Object-Relational Mapping (ORM) for PHP.
 
 [![Build Status](https://travis-ci.org/propelorm/Propel2.svg?branch=master)](https://travis-ci.org/propelorm/Propel2)
+[![Github actions Status](https://github.com/propelorm/Propel2/workflows/PropelORM%20CI/badge.svg?branch=master)](https://github.com/propelorm/Propel2/actions?query=workflow%3APropelORM%20CI+branch%3Amaster)
```

Comment:
Can we call the workflow just CI?
As we do for others as well?

Reply:
Updated.